src/vis_receive/workflow-test-vis-receive/test_vis_receive.py

```diff
@@ -0,0 +1,113 @@
+"""
+Example Vis Receive workflow
+"""
+
+# pylint: disable=C0103
+
+import logging
+import ska_sdp_config
+import distributed
+
+# Initialise logging
+logging.basicConfig()
+log = logging.getLogger('main')
+log.setLevel(logging.INFO)
+
+# Obtain connection to configuration database. This is the main way to
+# both obtain information about what the workflow is supposed to do,
+# and its way to request actions from the rest of the system (e.g. ask
+# for deployments of additional software)
+config = ska_sdp_config.Config()
+
+# Find processing block configuration from the configuration. Normally
+# this workflow should only get started once one is available, and we
+# just pull the ID from an environment variable here. But because the
+# infrastructure isn't there yet, for the moment workflows are started
+# speculatively, and wait until they spot a matching processing block
+# in the configuration.
+workflow = {
+    'id': 'testvisreceive',
+    'version': '0.0.1',
+    'type': 'realtime'
+}
+log.info("Waiting for processing block...")
+for txn in config.txn():
+
+    # Note that this process "claims" the workfow with a lease. This
+    # means that once a processing block has been claimed, this script
+    # must check in with the configuration database every ~10 seconds
+    # or will be declared dead (and presumably restarted). This
+    # obviously means that no serious work should actually happen here.
+    pb = txn.take_processing_block_by_workflow(
+        workflow, config.client_lease)
+    if pb is not None:
+        continue
+    txn.loop(wait=True)
+
+# Show
+log.info("Claimed processing block %s", pb)
+
+# Deploy Vis Receive with 1 worker.
+#
+# This is done by adding the request to the configuration database,
+# where it will be picked up and executed by appropriate
+# controllers. In the full system this will involve external checks
+# for whether the workflow actually has been assigned enough resources
+# to do this - and for obtaining such assignments the workflow would
+# need to communicate with a scheduler process. But we are ignoring
+# all of that at the moment.
+log.info("Deploying Vis Receive...")
+deploy_id = pb.pb_id + "-vis-receive"
+deploy = ska_sdp_config.Deployment(
+    deploy_id, "helm", {
+        'chart': 'skaorca/vis_receive:latest',
+        'values': {
+            'worker.replicas': 1,
+            # We want to access vis_receive in-cluster using a DNS name
+            'scheduler.serviceType': 'ClusterIP'
```

Comment:
Note that this would need to match the Helm chart's parameters as declared in `values.yml`. You might just want to keep it empty until you have something meaningful to put in - for instance this is where you would pass the location of the configuration database /if/ the visreceive execution engine needed it.